### DISCUSSION TITLE
Monthly ESMValTool community meeting November 2022

### DISCUSSION THREAD

**Discussion by bouweandela** [ESMValGroup/Community#60] [Meetings]
Dear @ESMValGroup/esmvaltool-developmentteam,

The next monthly ESMValTool community meeting will be in November. The result of the poll is that the meeting will take place on

Tuesday, November 15 at 4 PM CET

The duration will be one hour. [Join the zoom meeting](https://us02web.zoom.us/j/86356385817).

Calendar invitations to the meeting are sent to the mailing list, join it [here](https://docs.esmvaltool.org/en/latest/introduction.html#user-mailing-list).

Format and agenda

- Anyone with an interest in ESMValTool is welcome to join and ask questions
- We start the meeting by doing a round where everyone gets one minute to introduce themselves and tell what they will be working on in the next month or what they would like help with.
- Using the input from the introduction round, we choose some topics to talk about in more detail.
- If you already know that you would like to bring up a particular topic in the meeting, you can mention the [GitHub issue](https://github.com/ESMValGroup/ESMValTool/issues) on the topic in a comment below. That will make it easy to find for the other people in the meeting and we can already start discussing there before the meeting. Just [create a new GitHub issue](https://github.com/ESMValGroup/ESMValTool/issues/new) about your topic if there is no open issue yet.
- GitHub issues are our preferred way to communicate outside the meeting and keep track of work, so any feature requests/action items/etc discussed in the meeting should ideally result in a GitHub issue.

Notes from the previous meeting are available in https://github.com/ESMValGroup/Community/discussions/49#discussioncomment-3849977

**Comment by bouweandela** [comment]
@malininae @glpotter I also included some late timeslots to make it easier for people on your side of the ocean to participate, so vote for them if you're interested!

**Comment by bouweandela** [comment]
Thanks to everyone who filled out the doodle poll. It wasn't easy to find a time that works for many of us this time. I will send it around earlier next time to see if that works.

**Comment by valeriupredoi** [comment]
### ESMValTool Call 15 November 2022

#### Attendance: @glpotter @rswamina @remi-kazeroni @sloosvel @malininae @dhohn @TomasTorsvik @ehogan @catherinehardacre @pepcos @seyedmohammad19 @valeriupredoi 

#### Updates
- @rswamina  OBS work with @remi-kazeroni 
- @remi-kazeroni  discussion about CMORIZers
- @ehogan  cylc 8 on DKRZ but she will work on other projects in the near future
- @catherinehardacre was back in New Zealand but finding ESMValTool fans in NZ;
- @sloosvel CORDEX stuff, opened a bunch of PR's; 
- @pepcos work on CORDEX with Saskia; 
- @malininae working on a cool rapid attribution for extreme events recipe

#### Questions
- @glpotter: iris-xarray differences so data can be interchangeable (eg data produced with xarray read by iris), also CFSR cmorization - large volumes - @remi-kazeroni mentioned the strategy of OBS data hosting short term host OBS cmorized data locally on HPC sites, long term put it on ESGF
- @dhohn  new to ESMValTool (made a move from a Matlab env) but esmvaltool is awesome, but fixing data is a pain; Jupyter notebook to have a working env where diag gets run - @malininae has an ad-hoc solution if David asks via a Discussion/Issue - Ranjini recommends https://github.com/ESMValGroup/ESMValTool-JupyterLab/blob/main/README.md - so note here to start a Discussion item or an Issue
Ranjini: ESA-CCI BIOMAS data - how to do it, Remi/Birgit for obs4mips
- @malininae : rapid attribution for extreme events recipe - best way to handle a set of linear operations between datasets - either locally in one or more diagnostics, or ask for/contribute towards a set of preprocessors - probably preprocs are a better idea (given optimization), but again, a pointer to open an issue for discussion; also - how to regrid sea ice data (on irregular (tripolar?) grids) - issue to get the discussion rolling
- @seyedmohammad19 asked a couple questions via chat: how to get CMIP data into the Tool, and how to get station data into the Tool? We recommended the Tutorial, and also starting discussions/issues with specific problems, after getting used to the Tool a bit

Thanks to everyone who came along! Also, please edit these minutes if you want to add/detail stuff - am not a great minute taker :grin:

**Reply by bouweandela** [reply]
Thanks for hosting and for sharing the notes @valeriupredoi!

**Comment by malininae** [comment]
Opened issues https://github.com/ESMValGroup/ESMValTool/issues/2925 and https://github.com/ESMValGroup/ESMValTool/issues/2919 with my questions.
